tests/src/test/java/org/sonarqube/tests/issue/OrganizationIssueAssignTest.java

```diff
@@ -21,126 +21,143 @@
-import org.sonarqube.tests.Category6Suite;
 import org.sonarqube.ws.Issues;
 import org.sonarqube.ws.Issues.Issue;
-import org.sonarqube.ws.Organizations;
+import org.sonarqube.ws.Organizations.Organization;
+import org.sonarqube.ws.Projects.CreateWsResponse.Project;
+import org.sonarqube.ws.Qualityprofiles.CreateWsResponse.QualityProfile;
 import org.sonarqube.ws.Users.CreateWsResponse.User;
 import org.sonarqube.ws.client.issues.AssignRequest;
 import org.sonarqube.ws.client.issues.BulkChangeRequest;
 import org.sonarqube.ws.client.issues.SearchRequest;
-import org.sonarqube.ws.client.qualityprofiles.AddProjectRequest;
-import org.sonarqube.ws.client.projects.CreateRequest;
-import util.issue.IssueRule;
 
 import static java.lang.String.format;
 import static java.util.Collections.singletonList;
 import static org.assertj.core.api.Assertions.assertThat;
 import static util.ItUtils.expectHttpError;
 import static util.ItUtils.restoreProfile;
 import static util.ItUtils.runProjectAnalysis;
-import static util.ItUtils.setServerProperty;
 
 public class OrganizationIssueAssignTest {
 
   private final static String SAMPLE_PROJECT_KEY = "sample";
 
   @ClassRule
-  public static Orchestrator orchestrator = Category6Suite.ORCHESTRATOR;
+  public static Orchestrator orchestrator = OrganizationIssueSuite.ORCHESTRATOR;
   @Rule
   public Tester tester = new Tester(orchestrator);
 
-  @Rule
-  public IssueRule issueRule = IssueRule.from(orchestrator);
-
-  private Organizations.Organization org1;
-  private Organizations.Organization org2;
-  private User user;
-
-  @Before
-  public void setUp() {
-    org1 = tester.organizations().generate();
-    org2 = tester.organizations().generate();
-    user = tester.users().generate();
-    restoreProfile(orchestrator, getClass().getResource("/organization/IssueAssignTest/one-issue-per-file-profile.xml"), org1.getKey());
-  }
-
   @Test
   public void auto_assign_issues_to_user_if_default_assignee_is_member_of_project_organization() {
-    tester.organizations().addMember(org1, user);
+    Organization organization = tester.organizations().generate();
+    User user = tester.users().generateMember(organization);
+    provisionProjectAndAssociateItToQProfile(SAMPLE_PROJECT_KEY, organization);
+    tester.settings().setProjectSetting("sample", "sonar.issues.defaultAssigneeLogin", user.getLogin());
 
-    provisionProject(SAMPLE_PROJECT_KEY, org1.getKey());
-    setServerProperty(orchestrator, "sample", "sonar.issues.defaultAssigneeLogin", user.getLogin());
+    analyseProject(SAMPLE_PROJECT_KEY, organization);
 
-    analyseProject(SAMPLE_PROJECT_KEY, org1.getKey());
-
-    assertThat(issueRule.getRandomIssue().getAssignee()).isEqualTo(user.getLogin());
+    assertThat(getRandomIssue().getAssignee()).isEqualTo(user.getLogin());
   }
 
   @Test
   public void does_not_auto_assign_issues_to_user_if_default_assignee_is_not_member_of_project_organization() {
```

Comment:
to be exact: `do_not_auto_assign_issues_to_default_assignee_if_not_member_of_project_organization `